### PR TITLE
Troubleshooting about ignored Test & Adjust travis mongoDB for test.

### PR DESCRIPTION
**Pull request by lyw0149**
This PR makes :
- enable unit-test (Before this PR, travis test is working but don't execute any test).
- enable mongoDB unit-test

@ethkim @nlpenguin @djflexible @ifndefdeadmau5 @KaylaKwon  @SarahhhC

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -52,19 +52,23 @@ gulp.task('apidoc', (done) => {
...
-  return gulp.src('./dist-server/test/node/*.js')
+gulp.task('test:index', () => {
+  return gulp.src('./dist-server/test/index.js')
     .pipe(tape({
       reporter: faucet(),
     }));
 });
 
+gulp.task('test', () => {
+  runSequence('babel','test:index');
```
Comment (ethkim) [inline gulpfile.js]: So as we're adding more unit tests like test:index above, we'll need to link them all on this line? Could you add comment to do that if that's the case?

Also, weren't there more than index.js for tests? (user.js?) Before *.js was applied so all the tests were set up to run, but we now need to individually add them here.

Comment (lyw0149) [inline gulpfile.js]: index.js intergrate all js files in test/node folder and run all test. If you want to run test seperately, type like `tape test dist-server/test/node/user.js`

Comment (ethkim) [inline gulpfile.js]: Yup, I added those in the readme.

https://github.com/sv-bootcamp/yoda/tree/master/src/test/node

**Comment by ethkim** [comment]
LGTM go, go!

**Pull request merged**